src/AFM.ts

```diff
@@ -150,6 +151,45 @@ export namespace AFM {
         };
     }
 
+    export type ComparisonConditionOperator = 'GREATER_THAN'
+        | 'GREATER_THAN_OR_EQUAL_TO'
+        | 'LESS_THAN'
+        | 'LESS_THAN_OR_EQUAL_TO'
+        | 'EQUAL_TO'
+        | 'NOT_EQUAL_TO';
+
+    export interface IComparisonCondition {
+        comparison: {
+            operator: ComparisonConditionOperator
+            value: number;
+        };
+    }
+
+    export type RangeConditionOperator = 'BETWEEN' | 'NOT_BETWEEN';
+
+    export interface IRangeCondition {
+        range: {
+            operator: RangeConditionOperator;
+            from: number;
+            to: number;
+        };
+    }
+
+    export type MeasureValueFilterCondition = IComparisonCondition | IRangeCondition;
+
+    export interface ILocalIdentifierQualifier {
+        localIdentifier: string;
+    }
+
+    export type Qualifier = ObjQualifier | ILocalIdentifierQualifier;
+
+    export interface IMeasureValueFilter {
+        measureValueFilter: {
+            measure: Qualifier;
+            condition?: MeasureValueFilterCondition;
```

Comment:
How do we work with the measure value filter when it has no condition?

Reply:
Filters without condition will be ignored in gooddata-js